CommandLine/libEGM/gmk.cpp

```diff
@@ -455,22 +455,23 @@ int LoadSettings(Decoder &dec, Settings& set) {
 
   General* gen = set.mutable_general();
   Graphics* gfx = set.mutable_graphics();
+  Windowing *win = set.mutable_windowing();
   Info* inf = set.mutable_info();
 
   if (ver >= 800) dec.beginInflate();
-  gfx->set_start_in_fullscreen(dec.readBool());
+  win->set_start_in_fullscreen(dec.readBool());
   if (ver >= 600) {
-    gfx->set_smooth_colors(dec.readBool());
+    gfx->set_interpolate_textures(dec.readBool());
   }
-  gfx->set_window_showborder(!dec.readBool()); // inverted because negative in GM (e.g, "don't")
+  win->set_show_border(!dec.readBool()); // inverted because negative in GM (e.g, "don't")
   gen->set_show_cursor(dec.readBool());
-  gfx->set_window_scale(dec.read4());
+  gfx->set_view_scale(dec.read4());
   if (ver == 530) {
     dec.skip(8); // "fullscreen scale" & "only scale w/ hardware support"
   } else {
-    gfx->set_window_sizeable(dec.readBool());
-    gfx->set_window_stayontop(dec.readBool());
-    gen->set_color_outside_room_region(dec.read4());
+    win->set_is_sizeable(dec.readBool());
+    win->set_stay_on_top(dec.readBool());
```

Comment:
@JoshDreamland why did you rename this? I called it `window_stayontop` because the function is `window_set_stayontop` and I thought you wanted to match GML?
http://gamemaker.info/en/manual/405_10_window

Reply:
It reads sort of funny without the `window_` prefix, and we've already broken consistency all over the place for no real reason, but I'm not opposed to putting it back

Reply:
Your call boss, I just wanted to point it out 😛

Reply:
I think the underscores won't kill us. In particular, I think it's more intuitive to expect this grammar change, but if not, we can talk about helpful compatibility changes, later.